packages/sales-report-tool/index.ts

```diff
@@ -0,0 +1,91 @@
+import { getFormattedDate, getAmountOfNFTs } from "./utils";
+import { writeFileSync } from "fs";
+import { fetch } from "undici";
+
+interface Transaction {
+  number: number;
+  time: string;
+  txnId: string;
+  address: string;
+  amountOfSTX: number;
+  amountOfNFTsBought: number;
+  totalAmountOfNFTsSold: number;
+}
+
+// Array for storing transactions
+const arrOfTxn: Transaction[] = [];
+let total: number;
+// Minting wallet address
+const address = `SP2X0TZ59D5SZ8ACQ6YMCHHNR2ZN51Z32E2CJ173`;
+// Mint price
+const mintPrice = 100;
+
+const getAccounting = async () => {
+  await fetch(
+    `https://stacks-node-api.mainnet.stacks.co/extended/v1/address/${address}/transactions`
+  )
+    .then((response: any) => response.json())
+    .then((data: any) => {
+      total = data.total;
+    });
+  let indexOfLastTxn = total;
+  let index = 1;
+  let nftsSold = 0;
+
+  while (indexOfLastTxn > 0 && indexOfLastTxn <= total) {
+    await fetch(
+      `https://stacks-node-api.mainnet.stacks.co/extended/v1/address/${address}/transactions?offset=${
+        indexOfLastTxn - 20
+      }`
+    )
+      .then((response: any) => response.json())
```

Comment:
@pradel are there any ways around of getting types other than manually adding them by hand?

Reply:
The only way to get typed reponse would be to use https://github.com/hirosystems/stacks-blockchain-api/blob/master/client/README.md

Reply:
Should I use it or leave it w/o types here?

Reply:
I guess it's fine

Reply:
Good to merge then?